File: string/count-occurance-of-char-in-string.py
def count_occurrence_of_char_in_string(string, char):
    count = 0
    for i in range(0, len(string)):
        if string[i] == char:
            count += 1
    return count

if __name__ == "__main__":
    string = input("Enter the string: ")
    char = input("Enter the character to search: ")
    char_count = count_occurrence_of_char_in_string(string, char)
    print("The count of '%s' character in the string is: %s" % (char, char_count))

File: string/count-occurance-of-char-in-string.cpp
#include <iostream>
#include <string>

int count_occurrence_of_char_in_string(const std::string& str, char ch) {
    int count = 0;
    for (char c : str) {
        if (c == ch) {
            count++;
        }
    }
    return count;
}

int main() {
    std::string str;
    char ch;
    
    std::cout << "Enter the string: ";
    std::getline(std::cin, str);
    
    std::cout << "Enter the character to search: ";
    std::cin >> ch;

    int char_count = count_occurrence_of_char_in_string(str, ch);
    std::cout << "The count of '" << ch << "' character in the string is: " << char_count << std::endl;

    return 0;
}
